Update community-plugins.json

Torutu opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin:

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - []  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

Zachatoo commented:
[# DaggerHeart Obsidian Plugin](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/README.md#L1)
Naming something "Obsidian xyz" is reserved for first party products we create, please change this header.

[--corner: 10px;](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/style.css#L2-L7), [.card-inner {](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/style.css#L58-L84), [.input-field {](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/style.css#L121-L134), [.type-select {](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/style.css#L146-L156), [.title-small-padding {](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/style.css#L192-L203), [.field-row {](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/style.css#L224-L229), [.inline-label {](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/style.css#L212-L214), [.description-textarea {](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/style.css#L251-L256), [.flex-row {](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/style.css#L263-L274), most of your other classes
Please update your classes and custom properties to be more specific to your plugin to avoid conflicts with other plugins.

[.modal-title {](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/style.css#L216-L221)
Please don't overwrite core styling in a plugin, add another class/data attribute so it only applies to your plugin.

[export async function loadStyleSheet(plugin: Plugin) {](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/style.ts#L3-L19)
Obsidian will do this for you, you don't need to do this yourself.

["DaggerHeart Adversary Creator"](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/main.ts#L13), [`Load Tier ${tier} Adversaries`](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/main.ts#L21), ["Create Adversary Card"](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/main.ts#L28), ["Adversary Creator"](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversarySearch.ts#L19), ["Adversary Creator"](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversarySearch.ts#L37), ['Unnamed Adversary'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversarySearch.ts#L115), ['Adversary Stat Block'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L17), ['ATK Mod'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L116), ['Weapon Name'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L118), ['Weapon Range'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L119), ['Weapon Damage'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L120), ['Very Close'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L123), ['Very Far'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L123), ['Add Feature'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L131), ['Insert Card'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L140), ['Feature Name'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L169), ['Mark a Stress'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L180), ['Spend a Fear'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L180), ['Feature Description'](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryCreator.ts#L187)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

[this.addStatusBarItem().setText("Status Bar Text");](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/main.ts#L16)
Please remove this if you aren't using it.

[this.addRibbonIcon("venetian-mask", "DaggerHeart Adversary Creator", () => {](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/main.ts#L13-L15)
You should also add commands that activates the plugins functionality, the ribbon can be hidden and might be hidden by default in a future update.

---

[import adversariesTier1 from '../../adversaries/Adversaries-Tier-1.json';](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversarySearch.ts#L2-L5), [import adversariesTier1 from '../../adversaries/Adversaries-Tier-1.json';](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/adversaryList.ts#L2-L5)
I'm not seeing these files in your repo, please include them so I can review them.

[plugin.app.workspace.rightSplit.expand();](https://github.com/Torutu/daggerforge/blob/9adb7a83eac0e7c12bd5f7f14aa0060eb3261578/src/sidebar.ts#L14)
Prefer to use [`revealLeaf()`](https://docs.obsidian.md/Reference/TypeScript+API/Workspace/revealLeaf) instead, see <https://docs.obsidian.md/Plugins/User+interface/Views>

Torutu commented:
- Changed the header's name into "DaggerHeart Plugin"
- all my css classes now starts with **df** for daggerforge
- the modal-title is now **df-modal-title**, hopefully it doesn't overwrite the core style now
- removed **export async function loadStyleSheet(plugin: Plugin) {**
- the UI now uses Sentence case
- removed **this.addStatusBarItem().setText("Status Bar Text");**
- added a command for **this.addRibbonIcon("venetian-mask", "DaggerHeart Adversary Creator"**

---

- included the .json files in https://github.com/Torutu/daggerforge/tree/master/adversaries
- now using **revealLeaf()** instead
